src/import_roms.rs

```diff
@@ -37,6 +44,15 @@ pub fn subcommand<'a>() -> App<'a> {
                 .required(false)
                 .takes_value(true),
         )
+        .arg(
+            Arg::new("HASH")
+                .short('h')
```

Comment:
This conflicts with "-h" for the help menu.

Reply:
Thanks for the heads up. I had actually fixed that a while ago in the 0.11 branch but forgot to merge the changes to master :/ Did it earlier today.